test/src/js/index.spec.js

```diff
@@ -1,30 +1,25 @@
 'use strict';
 
-const { expect } = require('chai');
+
 
 const underTest = require('../../../src/js/index');
 
 describe('#init', function () {
-	it('init should be a Function', function () {
-		expect(underTest.init).to.be.a('function');
+	test('init should be a Function', function () {
+		expect(typeof underTest.init).toBe('function');
 	});
 
-	it.skip('should return undefined if the syndication flag is off and not call stubs', () => { });
-
-	context('when syndication customer', () => {
-		it.skip('should return undefined if the user isn’t migrated', () => { });
-		it.skip('should initialise the republising navigation', () => { });
-		it.skip('should end if the user isn’t allowed ft.com republishing', () => { });
-		it.skip('should initialise the rest if allowed ft.com republishing', () => { });
-	});
+	test.skip('should return undefined if the syndication flag is off and not call stubs', () => { });
```

Comment:
Am I correct that this is "those will need to be written later, but not really part of this ticket"?

Reply:
Yes. It's one of the reasons I got into this yak shaving excercise. There is no coverage on the init functionality so tricky making changes without writing test for existing features first. Thought I might as well switch up to jest if I'm gonna do that.